pkgmgmtutils/update_formulas.go

```diff
@@ -49,7 +49,9 @@ type FormulaStatus struct {
 	Err     error
 }
 
-func UpdateFormulas(projectRepoOwner string, projectRepoName string, parentPathSha256 string, fOpts []FormulaOptions) ([]FormulaStatus, error) {
+func UpdateFormulas(projectRepoOwner string, projectRepoName string, parentPathSha256 string, reShaFilenames string,
```

Comment:
nit: `projectRepoOwner, projectRepoName, parentPathSha256, reShaFilenames string`
You can leave out the types if they are the same